database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
```

Comment:
Observations: The database connection is established using procedural code, which can lead to issues with maintainability and scalability.

Recommendations:
- Use an object-oriented approach to manage the database connection. Create a `Database` class to encapsulate the connection logic.